src/main/java/com/gotech/accesscontrol/service/UserServiceImpl.java

```diff
@@ -41,18 +46,16 @@ public Response saveUser(UserRequest userRequest) {
     private String userValidation(User user) {
         String validationMessage = "";
         String validate = validateUser.isValid(user);
-        if (validate.equals(Constants.valid)) {
-            if (validateUser.isUserAlreadyExist(user).equals(Constants.empty)) {
-                serviceLogger.info(Constants.validMessage + user);
-                validationMessage = Constants.valid;
+        if (validate.equals(valid)) {
+            if (validateUser.isUserAlreadyExist(user).equals(empty)) {
+                serviceLogger.info(validMessage + user);
+                validationMessage = valid;
             } else {
-                serviceLogger.error(Constants.duplicate + user);
-                validationMessage = Constants.duplicate;
-                throw new UserAlreadyExistException(Constants.duplicate + user);
+                validationMessage = duplicate;
+                throw new UserAlreadyExistException(duplicate + user);
             }
         } else {
-            validationMessage = Constants.invalid + validate + user;
-            serviceLogger.error(validationMessage);
+            validationMessage = invalid + validate + user;
```

Comment:
Change this validation logic as told earlier refactor validate method remove multiple else